swfiles/@spinw/spinw.m

```diff
@@ -494,15 +494,11 @@
                     error('spinw:spinw:WrongInput', ...
                           'No spinw object could be found in the input figure');
                 end
-            end
-            
-            if isa(firstArg, 'spinw')
```

Comment:
Really minor but perhaps this should be a switch if all the conditions relate to only the `firstArg` (I think it's more readable but I don't really mind!)

Reply:
Ah good idea! My Python brain forgot switches exist

Reply:
Hmm, I don't think I can use a switch in this particular case, because the only way I can think of doing it is doing a `switch class(firstArg)`, which is fine for all cases except `ishandle`, where `class(firstArg) = 'matlab.ui.Figure'`. I could check for that class instead of using `ishandle` but am worried I'll change the behaviour in an unintended way...

Reply:
Fair enough, thanks for trying!